model/project_event.go

```diff
@@ -108,12 +108,33 @@ func (p *ProjectChangeEvents) RedactPrivateVars() {
 		if !isChangeEvent {
 			continue
 		}
+		modifiedPrivateVarKeys := findModifiedPrivateVars(changeEvent)
 		changeEvent.After.Vars = *changeEvent.After.Vars.RedactPrivateVars()
 		changeEvent.Before.Vars = *changeEvent.Before.Vars.RedactPrivateVars()
+		// If a private variable has been changed, change the before value of the
+		// variable to {REDACTED} so the event log will show that the private variable
+		// has been changed in the UI.
```

Comment:
Per [the style guide](https://docs.google.com/document/d/1-DB1FQ_zb5fX_0pnVcwyU0HmJPrloFBu6qSE8dkk8PU/edit#heading=h.l0w52tj2sr14) (under "Why is the code doing what it does"), instead of stating what this for loop is doing (which I think readers can deduce just from looking at it), can we instead explain why `RedactPrivateVars` isn't sufficient for redacting modified project vars in the change event's `Before` vars (and why this issue doesn't apply for the `After` vars)?

Based on the project vars unit test, it looks like the test will always show a blank value for the after value in a redacted var. Should the after value also be redacted (rather than empty)?

Reply:
Updated the comment. 

Switching the format to be "" --> {REDACTED} per this [other comment](https://github.com/evergreen-ci/evergreen/pull/7612#discussion_r1517922992)